Temporarly disable Travis CI  Mac OS X

G10h4ck opened the pull request.

G10h4ck committed:
Support Mac OS X 10.13 as build target

Attempt to fix travis CI macosx

G10h4ck committed:
Mac OS X fix ordered comparison between pointer and zero

G10h4ck commented:
Travis CI has updated to Mac OS X 10.13 the new compiler complain a lot about RetroShare code...

G10h4ck commented:
this may be related https://github.com/termux/termux-packages/issues/1149

sehraf commented on retroshare.pri:
```diff
@@ -500,6 +501,12 @@ macx-* {
 	}
 
+    rs_macos10.13 {
+        message(***retroshare.pri: Set Target and SDK to MacOS 10.12 )
```
nit: 13 instead of 12

G10h4ck committed:
Fix trivial copy paste error

G10h4ck committed:
Temporarly disable travisCI mac os x building

While we figure out how to fix the building on Max OS X 10.13 disable
travis CI on Mac OS X to avoid false broken CI on PR

PhenomRetroShare commented:
Already worked on it with https://github.com/PhenomRetroShare/RetroShare/tree/Fix_TravisMacOSCompil. But get blocked to Compil error with unordered_map:
https://travis-ci.org/PhenomRetroShare/RetroShare/jobs/411227842#L1340

G10h4ck commented:
yeah it's same error here also i have found more probably related reports on other projects

https://github.com/sass/libsass-python/issues/240
